package-lock.json

```diff
@@ -0,0 +1,6550 @@
+{
```

Comment:
Should we remove this file since we already have a `yarn.lock` file?

Reply:
Oh yeah, my bad :)